scipy/stats/tests/test_discrete_distns.py

```diff
@@ -101,6 +102,79 @@ def test_betabinom_bernoulli():
     assert_almost_equal(p, expected)
 
 
+def test_issue_10317():
```

Comment:
This test does demonstrate that gh-10317 is resolved for SciPy stats, but it is still open for `scipy.special`.